tgstation.dme

```diff
@@ -2360,6 +2359,7 @@
 #include "code\modules\mob\living\carbon\human\species.dm"
 #include "code\modules\mob\living\carbon\human\status_procs.dm"
 #include "code\modules\mob\living\carbon\human\suicides.dm"
+#include "code\modules\mob\living\carbon\human\monkey\monkey.dm"
```

Comment:
Do we need an entire folder for this one file?

Reply:
i figured itd be nicer than clutter the human folder

Reply:
future gorillas, orangos, baboons and chimpanzee place holders